docs/sdkx-cocos2dx/migration-guide.mdx

```diff
@@ -0,0 +1,635 @@
+---
+sidebar_position: 150
+title: Migration Guide
+description: "SDK X Cocos2dx plugin is a Hybrid SDK that helps you roll out innovation faster by enabling most of the updates over the air that flow to end-users without any downtime or app updates. This migration guide will walk you through the steps you need to take to migrate from the Helpshift legacy SDK cocos2dx plugin to the SDK X cocos2dx plugin."
+---
+
+import {
+  Admonition,
+  Tabs,
+  TabItem,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Migration Guide
+
+<Intro>
+
+SDK X Cocos2dx plugin is a Hybrid SDK that helps you roll out innovation faster by enabling most of the updates over the air that flow to end-users without any downtime or app updates. This migration guide will walk you through the steps you need to take to migrate from the Helpshift legacy SDK Cocos2dx plugin to the SDK X Cocos2dx plugin.
+
+</Intro>
+
+## Steps to migrate to SDK X Cocos2dx plugin {#steps-to-migrate}
+
+<Steps title="Migrate from legacy SDK Cocos2dx plugin to SDK X Cocos2dx plugin">
+  <Step
+    number="1"
+    title="Check migration support"
+    subtitle="Go through migration guide"
+    link="/sdkx-cocos2dx/migration-guide-android/#unsupported-features"
+    newTab={true}
+  />
+  <Step
+    number="2"
+    title="Update to SDK X Cocos2dx plugin"
+    subtitle="Migrate code/features step by step"
+    link="/sdkx-cocos2dx/migration-guide-android#update-helpshift-dependencies"
+    newTab={true}
+  />
+  <Step
+    number="3"
+    title="Test the migration"
+    subtitle="Validate the functionality"
+  />
+</Steps>
+
+## Unsupported features {#unsupported-features}
+
+We recommend to hold off on SDK X migration if you are currently using the following APIs/features -
+
+- [Guided issue filing](/cocos2d-x/support-tools-android/#dynamic-forms)
+- [FAQ filtering by tags](/cocos2d-x/support-tools-android/#faq-filtering)
+- [Minimum Issue description length](/cocos2d-x/sdk-configuration-android/#minimum-issue-desc-length)
+- [SDK UI strings customization](/cocos2d-x/string-customization-android/)
+- [Switching between light and dark themes](/cocos2d-x/design-android/#skinning)
+
+These features are yet to be developed, and we may add them to our future roadmap. You can reach out to [support@helpshift.com](mailto:support@helpshift.com) to raise any query for the above.
+
+## Supported SDK Versions {#supported-sdk-versions}
+
+A successful migration indicates maintaining users’ data and chat history when they move from one SDK version to another. We support the migration of the SDK Cocos2dx plugin from version 4.x+ to the SDK X Cocos2dx plugin.
```

Comment:
We should try to clearly mention that 4.x is Legacy SDK Cocos2dx plugin version and not Cocos2dx version itself.